docs/wiki/99_Developers/Developers_Guide.md

```diff
@@ -184,19 +185,19 @@ First, make a remote of their fork:
           
 Then, fetch their remote fork and checkout the branch related to their PR:
 
-    git fetch alex
-    git checkout alex/2020-05-06-something
+    git fetch remote_name
```

Comment:
is `remote_name` the standard way we refer to this across all the install docs?  consistency is super important here.

Reply:
There's no other place in the public docs on Github that mentions forks/remote.
In the latest commit, I made sure everything in this doc was phrased in the same way across sections ("my_fork", "branch_name", etc)